src/components/FormLabel/styles/FormLabel.css.ts

```diff
@@ -3,7 +3,43 @@ import styled from '../../styled'
 
 export const FormLabelUI = styled('div')`
   ${baseStyles};
+  display: flex;
+  flex-flow: ${props => (props.inline ? 'row' : 'column')};
+  justify-content: space-between;
+  align-items: ${props => (props.inline ? 'flex-end' : 'stretch')};
```

Comment:
@tinkertrain Although Emotion does support prop interpolation.. I actually prefer just using CSS className.modifiers for the rules, example:

```
flex-flow: column;

  &.is-inline {
    flex-flow: row;
  }
```

I personally find it easier to parse 😊 (and it's usually less verbose. Usually).

However, for trickier things that involve dynamic calculation, like your `calculateContentRules` implementation, it's okay.

What you think? :D

Reply:
I like it! In my first revision I didn't have the `is-inline` class added so I couldn't do that, but now I do so I will update this :)